packages/shared/src/features/profile/components/ProfileSkills.tsx

```diff
@@ -0,0 +1,179 @@
+import React, { useRef, useState } from 'react';
+import type { ReactElement } from 'react';
+import type { PopoverContentProps } from '@radix-ui/react-popover';
+import { Popover, PopoverAnchor } from '@radix-ui/react-popover';
+import { Controller, useFormContext } from 'react-hook-form';
+import { useQuery } from '@tanstack/react-query';
+import { TextField } from '../../../components/fields/TextField';
+import { FeedbackIcon, SearchIcon } from '../../../components/icons';
+import { IconSize } from '../../../components/Icon';
+import { TagElement } from '../../../components/feeds/FeedSettings/TagElement';
+import { PopoverContent } from '../../../components/popover/Popover';
+import useDebounceFn from '../../../hooks/useDebounceFn';
+import { GenericLoaderSpinner } from '../../../components/utilities/loaders';
+import {
+  Typography,
+  TypographyType,
+} from '../../../components/typography/Typography';
+import { getKeywordAutocompleteOptions } from '../../opportunity/queries';
+
+type ProfileSkillsProps = {
+  name: string;
+};
+
+const ProfileSkills = ({ name }: ProfileSkillsProps): ReactElement => {
```

Comment:
djeez this is a thing XD, hard to validate, likely we just test the outcome rather.